content/project/resistance-lab/index.md

```diff
@@ -12,7 +12,7 @@ image:
 imagealt:
 description: "How we worked with Resistance Lab from the start to build their first website and support them with branding and design"
 weight: 2
-draft: true
+draft: false
```

Comment:
other PR isn't merged in yet is this ok?

Reply:
oh good catch! Must have done that in error. Will revert!